src/openforms/formio/components/translations.py

```diff
@@ -7,12 +7,23 @@ def translate_options(
     enabled: bool,
 ) -> None:
     for option in options:
+        if "openForms" not in option:
+            continue
+
         if not (translations := option.get("openForms", {}).get("translations")):
             continue
 
-        translated_label = translations.get(language_code, {}).get("label", "")
-        if enabled and translated_label:
-            option["label"] = translated_label
+        if enabled:
+            translated_label = translations.get(language_code, {}).get("label", "")
+            translated_description = translations.get(language_code, {}).get(
+                "description", ""
+            )
+
+            if translated_label:
+                option["label"] = translated_label
+
+            if translated_description:
+                option["description"] = translated_description
```

Comment:
Simplified (I think)

```suggestion
        if enabled and (_translations := translations.get(language_code)):
            if (translated_label := _translations.get("label")):
                option["label"] = translated_label
                
            if (translated_description := _translations.get("description")):
                option["description"] = translated_description
```